packages/fxa-content-server/app/scripts/templates/partial/sync-suggestion.mustache

```diff
@@ -1,6 +1,6 @@
 {{#showSyncSuggestion}}
   <div class="info mx-auto mb-5" id="suggest-sync">
       {{#unsafeTranslate}}Looking for Firefox Sync? <a %(escapedSyncSuggestionAttrs)s>Get started here</a>{{/unsafeTranslate}}
-      <span class="dismiss" tabindex="-1">&#10005;</span>
+      <img src="/images/closed.svg" class="dismiss" tabindex="-1" alt="&#10005;" />
```

Comment:
And just because my cursed eyes just saw this, I noticed we have slightly different file names here and above ("close" vs "closed"):
```html
<img src="/images/close.svg" class="dismiss" tabindex="-1" alt="&#10005;" />
<img src="/images/closed.svg" class="dismiss" tabindex="-1" alt="&#10005;" />
```

And per the following search, I think _this_ reference is wrong:

```sh
git ls-files packages/fxa-content-server | grep -E "/closed?.svg"

packages/fxa-content-server/app/images/close.svg
```

Reply:
Ah, yes... that was a relic from testing the image fallback....